include/abb_librws/rws_interface.h

```diff
@@ -419,6 +419,13 @@ class RWSInterface
    */
   StaticInfo collectStaticInfo();
 
+  /**
+   * \brief Retrieves the configuration instances for the present robots.
```

Comment:
What do you mean by *present*? Would that be the robots connected to/configured on the controller the current RWS session communicates with?

If so, that might be a better description of what this method does.

Reply:
Opps, `present robots` should be `robots defined in the system`.

Fixed in 42587d6